src/views/primary-auth/CustomButtons.js

```diff
@@ -34,13 +34,18 @@ define([
     children: function () {
       var children = [],
           socialIdpButtons = this.settings.get('configuredSocialIdps'),
+          pivButton = this.settings.get('piv'),
           customButtons = this.settings.get('customButtons'),
           divider = dividerTpl({text: Okta.loc('socialauth.divider.text', 'login')});
 
       _.each(socialIdpButtons, function (button) {
         children.push(this._createSocialIdpButton(button));
       }, this);
 
+      if (this.settings.get('hasPivCard')) {
```

Comment:
wondering if PIV should be above social idps? We do the PIV sign in, instead of social idps that redirect to other page... can you double check with design or pm?

Reply:
@vincentvoong-okta do you have a preference? PIV and Social buttons together is very rare.. but just in case.

Reply:
@mauriciocastillosilva-okta @gowthamidommety-okta 
I don't know enough about this edge case to have a very informed decision but I'd imaging if a user has the choice between PIV/CAC and social auth, PIV/CAC should have priority.

Reply:
thanks Vince! Will move it up